hugegraph-store/hg-store-dist/src/assembly/ext-lib/hugegraph-core-1.5.0.pom

```diff
@@ -0,0 +1,441 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements. See the NOTICE file distributed with this
+  work for additional information regarding copyright ownership. The ASF
+  licenses this file to You under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations
+  under the License.
+  -->
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
```

Comment:
In accordance with the hierarchical structure, the parent of `hugegraph-core` should be `hugegraph-server`. However, due to existing discrepancies between the community version of `hugegraph-server` and the internal version (such as `revision`), a temporary adjustment has been made for the purpose of building.

Reply:
> In accordance with the hierarchical structure, the parent of `hugegraph-core` should be `hugegraph-server`. However, due to existing discrepancies between the community version of `hugegraph-server` and the internal version (such as `revision`), a temporary adjustment has been made for the purpose of building.

Update: miss the msg before, and we choose use a new version for all modules